### DISCUSSION TITLE
Application approved but mail not received

### DISCUSSION THREAD

**Discussion by atakanapaydin** [community/community#129949] [GitHub Education]
### Hi everyone,

Hi everyone,
My education verification was approved three days ago, but I have not received any emails about my benefits. Is there anyone to help about it?
![Screenshot_1](https://github.com/community/community/assets/66443801/0a318792-a8ca-4b8c-8cf0-ec7cc5383187)

**Comment by Sworzen1** [comment]
Hello @atakanapaydin 

Look on similar [Thread](https://github.com/orgs/community/discussions/129836)

1. Check Spam/Junk Folder: Sometimes, emails can end up in the spam or junk folder. Make sure to check these folders in your email account.

2. Email Confirmation: Ensure that the email address associated with your GitHub account is correct and that you have confirmed it.

3. GitHub Support: You can also contact GitHub support directly through the [GitHub Support](https://support.github.com/).
